projectional/src/main/java/jetbrains/jetpad/projectional/cell/BaseProjectionalSynchronizer.java

```diff
@@ -66,6 +70,8 @@
   private DeleteHandler myDeleteHandler = DeleteHandler.EMPTY;
   private ContentKind<SourceItemT> myItemKind;
   private Function<SourceItemT, SourceItemT> myCloner;
+  private ListMap<ContentKind, ContentToItemMapper> myCompatibleContentKinds = new ListMap<>();
```

Comment:
`myContentKind` sounds like something that this editor yields, but actually it's something an editor can fetch items from. (see below)